ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -267,7 +267,7 @@ def __init__(
         self.is_continuous_int = torch.nn.Parameter(
             torch.Tensor([int(act_type == ActionType.CONTINUOUS)])
         )
-        self.act_size_vector = torch.nn.Parameter(torch.Tensor(act_size))
+        self.act_size_vector = torch.nn.Parameter(torch.Tensor([sum(act_size)]))
```

Comment:
```suggestion
        self.act_size_vector = torch.nn.Parameter(torch.Tensor([sum(act_size)]), requires_grad=False)
```

Reply:
should this be a `buffer`?

Reply:
I remember I left it as parameter due to some errors I saw during export using buffer. Now sure if it's still the case.